### ISSUE TITLE
Merge with eslint-plugin-ember

### ISSUE THREAD

**Issue by Turbo87** [ember-best-practices/eslint-plugin-ember-best-practices#73]
I don't know why this project was started when we also have https://github.com/netguru/eslint-plugin-ember but I think having two projects that try to do similar things is just wasting resources.

I would like to propose to merge this project with https://github.com/netguru/eslint-plugin-ember under a shared GitHub org so that everyone can act as a maintainer.

see https://github.com/netguru/eslint-plugin-ember/issues/88

**Comment by rwjblue** [comment]
This project started prior to the net guru project being renamed and popularized. I'm all for a shared collaboration in a repo with shared ownership. Happy to host that in this org (the org name seems to fit the purpose) if others are also 👍.

**Comment by scalvert** [comment]
I'm all for merging. I also think it makes sense to merge these rules into the netguru ones, and ultimately move that project under this org. We can all act as maintainers of the newly merged one.

**Comment by sandstrom** [comment]
Awesome to see that the two efforts are (or will be) combined! 🥇 

What's the state of this merge?

It seems like the netguru repo has been moved to `https://github.com/ember-cli/eslint-plugin-ember`, but this one also seems to be active.

If eslint-plugin-ember has superseded this one, how about adding a large warning/note on the readme in this repo (I accidentally added this one to our project, before realising that a similar plugin existed under the `ember-cli` org).

**Comment by Turbo87** [comment]
> If eslint-plugin-ember has superseded this one, how about adding a large warning/note on the readme in this repo

I think that would be a good idea. Feel free to open a PR for this :)

**Comment by sandstrom** [comment]
@Turbo87 Happy to submit a PR.

@rwjblue Would you like me to PR a readme-change pointing people towards [eslint-plugin-ember](https://github.com/ember-cli/eslint-plugin-ember)?

**Comment by trentmwillis** [comment]
I'm unsure about the other maintainers, but I never received any real notice of the merge occurring or being completed. If it has, then we likely have diverged again as there has been ongoing work in this repo.

**Comment by rwjblue** [comment]
Agreed. We need to make a meta issue to evaluate and migrate or deprecate (with guidance) each individual rule. 

A deprecation notice is a tad premature until we also provide guidance...
